src/Microsoft.ServiceFabric.Actors/Diagnostics/ActorMethodDiagnosticData.cs

```diff
@@ -12,8 +12,16 @@ internal struct ActorMethodDiagnosticData
     {
         internal ActorId ActorId;
         internal long InterfaceMethodKey;
-        internal TimeSpan? MethodExecutionTime;
         internal Exception Exception;
         internal RemotingListenerVersion RemotingListener;
+        internal TimeSpan? MethodExecutionTime;
+
+        internal ActorMethodDiagnosticData(ActorId actorId, long interfaceMethodKey, Exception exception, RemotingListenerVersion remotingListener)
+        {
```

Comment:
It's odd that we don't have the method execution time with the exception. I could see them being mutually exclusive, but there's value in measuring the execution time even when an exception is thrown.